packages/grafana-ui/src/components/PanelContainer/PanelContainer.tsx

```diff
@@ -0,0 +1,23 @@
+import { css, cx } from '@emotion/css';
+import React, { DetailedHTMLProps, HTMLAttributes } from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+
+import { useStyles2 } from '../../themes';
+
+type Props = DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>;
+
+const getStyles = (theme: GrafanaTheme2) =>
```

Comment:
Nit: Can we have the styles declared after the component, so it's consistent with other components?